fastchat/serve/openai_api_server.py

```diff
@@ -11,6 +11,7 @@
 import argparse
 import json
 import logging
+import httpx
```

Comment:
Why is this still required?

Reply:
I found one remaining usage of httpx. Should this be updated?
https://github.com/lm-sys/FastChat/blob/cbbd682692f98118574f3f463d617c3515301a86/fastchat/serve/openai_api_server.py#L580

Reply:
> I found one remaining usage of httpx. Should this be updated?
> 
> https://github.com/lm-sys/FastChat/blob/cbbd682692f98118574f3f463d617c3515301a86/fastchat/serve/openai_api_server.py#L580

Here httpx is used as stream, which is a different usage with other place.
I am not sure which aiohttp api should be used here, and how to verify it...

Reply:
httpx is not a built-in library. Please move it to the second block of import statement.